pyccel/ast/literals.py

```diff
@@ -36,7 +36,12 @@ class LiteralTrue(sp_BooleanTrue, Literal):
     def __init__(self, precision = default_precision['bool']):
         self._precision = precision
 
-    def set_precision(self, precision):
+    @property
+    def precision(self):
+        return self._precision
```

Comment:
This function should be inherited from `PyccelAstNode`

Reply:
but ```
@precision.setter
def precision(self, new_precision)
``` won't work without it.

Reply:
ah, or i can inherit `precision` from `PyccelAstNode`.